service/controller/v1/resource/app/fixture_app_cr_test.go

```diff
@@ -0,0 +1,64 @@
+package app
+
+import (
+	"time"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
+	applicationv1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func newTestAppCRFromFilled(modifyFunc func(*applicationv1alpha1.App)) *applicationv1alpha1.App {
```

Comment:
This thing here follows the approach from `e2etemplates` repo.

I'm thinking about extracting top-level `fixture` package which we could easily copy around our projects.